lasagne/tests/layers/test_merge.py

```diff
@@ -102,6 +110,8 @@ def crop_test(cropping, inputs, expected):
             crop_test(crop_bad, [x0[:, :, :, 0], x1, x2[:, :, :, :, None]],
                       [x0[:1, :2, :, :], x1[:1, :2, :, :], x2[:1, :2, :, :]])
 
+        num_inputs_larger_than_ndim_input()
```

Comment:
Wouldn't it be easier to just add another `crop_test()` call above such as:
```python
crop_test(crop_x, [x0, x1, x2, x0, x1, x2],
          [x0[:1, :2, :, :], x1[:1, :2, :, :], x2[:1, :2, :, :],
           x0[:1, :2, :, :], x1[:1, :2, :, :], x2[:1, :2, :, :]])
```
This avoids adding another function.
```